project/lesson4/final/index.html

```diff
@@ -21,7 +21,7 @@
   <link rel="stylesheet" href="http://weloveiconfonts.com/api/?family=zocial" />
 
   <!-- loads the Udacity Feedback extension -->
-  <meta name="udacity-grader" content="http://udacity.github.io/responsive-images/project/project-grader.json" unit-tests="http://udacity.github.io/responsive-images/project/project-grader.js">
+  <meta name="udacity-grader" content="project-grader.json" unit-tests="project-grader.js">
```

Comment:
The problem has to do with the origin. Since it's not the same origin as the document, AMO did complain about that.

I too think it's not the best thing to include all the test files inside the ZIP file.